.github/workflows/tag.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Build and push Docker Image on Tag
+
+on:
+  push:
+    tags:
+      - '[0-9]*'
+
+jobs:
+  build_and_push_docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Could you please change it to `v3`? An example: https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/.github/workflows/tag.yml#L16 .